Makefile

```diff
@@ -66,8 +63,7 @@ validate:
 
 .PHONY: test
 test:		## Runs all the tests
-# TODO: add a test to ensure we are not missing targets here
-test: test-unit test- test-infection
```

Comment:
Not sure when `test-e2e` has been changed to `test-` but in any case, at least now the MakefileTest should detect this kind of error :)

Reply:
Looks like here https://github.com/infection/infection/commit/8b5fe199e222eff7a0fd5960815c91f825b0489f#diff-b67911656ef5d18c4ae36cb6741b7965L70

Reply:
So myself... which further proves that I am the main danger to myself and that I cannot code without testing everything

Reply:
Also this might be the reason e2e tests took so much time.

Reply:
Most likely :D